package.json

```diff
@@ -24,7 +24,8 @@
     "version": "bash scripts/version.sh && git add .",
     "version-nightly": "bash scripts/should-nightly-run.sh && npm version prerelease --no-git-tag-version --preid=$(git rev-parse HEAD)",
     "postversion": "bash scripts/push-if-git.sh",
-    "bundlesize": "yarn build && yarn build:dead-code-elimination && size-limit"
+    "bundlesize": "yarn build && yarn build:dead-code-elimination && size-limit",
+    "chromatic": "chromatic --project-token=hekccxx8mtl --exit-zero-on-changes"
```

Comment:
Can we move this hardcoded token to secrets `--project-token=hekccxx8mtl`?
Ref: https://www.chromatic.com/docs/ci

Reply:
definitely, i'll regen it.